### ISSUE TITLE
Default Config Location

### ISSUE THREAD

**Issue by HermanGaviriaMarin** [jordanruthe/KlipperScreen#528]
Grandmaster Jordanruthe, 

I'm Herman, student at USF and 3D printer enthusiast. I'm currently running klipperscreen and its been great, thank you for all the hard work and for making this awesome feature. 

On to the reason why I'm making this new issue. When I first set up my klipperscreen the first thing I did was type on the google browser "klipperscreen" then the first thing I opened was [[](url)](https://klipperscreen.readthedocs.io/en/latest/) this of course guides the new users really well but there is one key mistake or point where the new users may find themselves in trouble. The short template that is left for the user to copy and paste to create their klipperscreen.config is not complete as shown below (this is just one of the sections, I know there is more but for the sake of having one screen shot this is what i chose to show)

https://klipperscreen.readthedocs.io/en/latest/Configuration/#:~:text=and%20restart%20KlipperScreen.-,Include%20files,define%20custom%20systemctl%20command%20for%20restart%20like%20xrdp%0Aservice%3A%20KlipperScreen,-Printer%20Options
![image](https://user-images.githubusercontent.com/84948613/155939994-87c4f388-e631-439b-9504-249a0fa22b87.png)

With these quick little templates that one can copy and paste the user will undoubtedly be able to get their klipperscreen working, I know this because I did it this way and it worked. But the issue at hand is that they will not have all the features. They will be able to home their printer, move their xyz and print. But all of the other bells and whistles that are found in https://github.com/jordanruthe/KlipperScreen/blob/master/ks_includes/defaults.conf will be unavailable. 

I was able to find that template but only after a lot of searching around. It would be nice if you could leave a link in the main website to that template, or to simply include it there for easy finding. I know that many of the klipper users are veterans when it comes to klipper and all of this firmware magic but me personally and many others aren't. Making things a little simpler for those noobies like myself helps a lot. 
![image](https://user-images.githubusercontent.com/84948613/155940458-3a233807-5675-4fe9-92ac-7604e347365a.png)

This config file is the one people need. You could just simply leave a little hyperlink in the instructions or something or say that the config the users need is located in that folder.

**Comment by alfrix** [comment]
I disagree the config is explained as the config reference in Klipper, and when an option is not listed there, it's because it's on the settings panel
